packages/fxa-settings/src/pages/Signup/ConfirmSignupCode/mocks.tsx

```diff
@@ -51,7 +51,6 @@ export function createMockOAuthIntegration(
     getRedirectUri: () => MOCK_REDIRECT_URI,
     getService: () => serviceName,
     wantsTwoStepAuthentication: () => false,
```

Comment:
Is this mock missing isOAuth?

Reply:
It was, I hadn't fully mocked everything out when I added that `isOAuth` method in case it's not what we wanted, and it wasn't. I'm much happier with what we've got now thanks to Ben's comment + yours.